### PR TITLE
feat(Icon): Icons for screen sharing control in the calling screen

### PR DESCRIPTION
**Pull request by jay-howe**
Problem description
Please port the new ubar icons for screen sharing control to StarDust
image

### CODE REVIEW

**Comment by alinais** [comment]
#1485

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-control-request.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <path
+        d="M12.125,20.4066315c0.2761421,0,0.5,0.2238579,0.5,0.5s-0.2238579,0.5-0.5,0.5H9.5
```
Comment (alinais) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-control-request.tsx:8]: Isn't the size of this icon too small as a default one? I was looking into the screener.io catches and it seems to be half sized compared to the `call-control-release` one

Comment (alinais) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-control-request.tsx:8]: image

Comment (jay-howe) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-control-request.tsx:8]: Thanks for pointing out. I had it fixed in one file, but not the other.

**Comment by kuzhelov** [comment]
@codepretty, could you, please, take a look and verify SVG paths/viewport size/name for the introduced icon? Thank you!

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 2]
```diff
@@ -150,6 +152,8 @@ export default {
   'call-parking': callParking,
   'call-video': callVideo,
   'call-video-off': callVideoOff,
+  'call-control-release': callControlRelease,
```
Comment (kuzhelov) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:155]: lets ensure those are pasted in alphabetical order to the set of other ones

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 3]
```diff
@@ -12,6 +12,8 @@ import bullets from './bullets'
 import calendar from './calendar'
 import call from './call'
 import callParking from './callParking'
+import callControlRelease from './callControlRelease'
```
Comment (kuzhelov) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:15]: lets maintain alphabetical order, to simplify search scenarios

Comment (jay-howe) [reply packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:15]: I am keeping release and request as a group (viewer) as they are most closely related. The other call-control-xxx icons are a different group (sender)

Comment (jay-howe) [reply packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:15]: I have grouped them by concepts as opposed to lexical order.

Comment (jay-howe) [reply packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:15]: The existing list is already out of order. Do you want me to fix it?

Comment (kuzhelov) [reply packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:15]: no, lets stick to the order pattern already established -  thank you for clarification!

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts (hunk 2 above)
Comment (kuzhelov) [reply packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:155]: agreed to keep 'concepts-based' order

**Review by kuzhelov — APPROVED** [review]

**Review by kuzhelov — CHANGES_REQUESTED** [review]
please, just add changelog entry

**Review by kuzhelov — APPROVED** [review]

**Pull request merged**